sdk/appconfiguration/app-configuration/src/policies/throttlingRetryPolicy.ts

```diff
@@ -10,23 +10,29 @@ import {
   WebResource,
   HttpOperationResponse,
   Constants,
-  RestError
+  RestError,
+  RetryOptions
 } from "@azure/core-http";
 import { delay } from "@azure/core-http";
 
 /**
  * @internal
  */
-export function throttlingRetryPolicy(): RequestPolicyFactory {
+export function throttlingRetryPolicy(
+  retryOptions?: Pick<RetryOptions, "maxRetries" | "maxRetryDelayInMs">
```

Comment:
update the parameter type to use the local one?

Reply:
updated!